paper-pnas.tex

```diff
@@ -0,0 +1,219 @@
+\documentclass[9pt,twocolumn,twoside,lineno]{pnas-new}
+% Use the lineno option to display guide line numbers if required.
+% Note that the use of elements such as single-column equations
+% may affect the guide line number alignment. 
+
+\templatetype{pnasresearcharticle} % Choose template 
+% {pnasresearcharticle} = Template for a two-column research article
+% {pnasmathematics} = Template for a one-column mathematics article
+% {pnasinvited} = Template for a PNAS invited submission
+
+\title{How and Why to run a Hack Week}
+
+% Use letters for affiliations, numbers to show equal authorship (if applicable) and to indicate the corresponding author
+\author[a,b,1]{Daniela Huppenkothen}
+\author[c,1,2]{Anthony Arendt} 
+\author[b,d]{David W. Hogg}
+\author[e]{Karthik Ram}
+\author[f]{Jake VanderPlas}
+\author[g]{Ariel Rokem}
+
+\affil[a]{Center for Data Science, New York University, 65 5th Avenue, 7th Floor, New York, NY 10003, USA}
+\affil[b]{Center for Cosmology and Particle Physics, New York University, 726 Broadway, 10th Floor, New York, NY 10003, USA}
+\affil[c]{}
+\affil[d]{Center for Computational Astrophysics, Flatiron Institute, 162 5th Ave, New York, NY 10010, USA}
+\affil[e]{}
+\affil[f]{}
+\affil[g]{}
+
+% Please give the surname of the lead author for the running footer
+\leadauthor{Huppenkothen} 
+
+% Please add here a significance statement to explain the relevance of your work
+\significancestatement{As scientific disciplines grapple with more data sets of rapidly increasing complexity and size, new approaches are urgently required to introduce new statistical and computational tools into research communities and improve the cross-disciplinary exchange of ideas. In this paper, we introduce a new type of scientific workshop, called a \textit{hack week}, which allows for fast dissemination of new methodologies into scientific communities, and fosters exchange and collaboration within and between disciplines. We present implementations of this concept in astronomy, neuroscience and geoscience, and show that hack weeks produce positive learning outcomes, foster lasting collaborations, yield scientific results and improve attitudes toward open science and reproducibility.}
```

Comment:
Super!

Reply:
I feel like this should emphasize that this is useful for *all* sciences, but I couldn't quite put that into words. We have ~ 15 words left to the word limit, so if someone wants to put this into an even broader context, be my guest. :)